index.html

```diff
@@ -7,33 +7,241 @@
   <link rel="stylesheet" href="styles.css">
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@400&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@400;600;800&display=swap" rel="stylesheet">
   <title>Rhodny's Porfolio Template</title>
 </head>
 <body>
-  <header>
-    <nav class="top-edge-container">
-      <div class="geometry-image"></div>
-      <div class="ic-menu"></div>
-    </nav>
-  </header>
-  <section class="section1-container">
-    <div class="section1">
-      <h1 class="name dark-blue">Charlie Giorgania</h1>
-      <div class="contents">
-        <p class="intro-p dark-blue">
-          Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work.
-        </p>
-        <p class="intro-p dark-blue">
-          If you like what you see and have a project you need coded, don't hesitate to contact me.
-        </p>
-        <button class="start-collab bg-dark-blue">Start collaboration</button>
+  <!--Headline section-->
+  <div>
+    <header>
+      <nav class="top-edge-container">
+        <div class="geo-background-img"></div>
+        <div class="ic-menu"></div>
+      </nav>
+    </header>
+    <section class="section-container">
+      <div class="section-sub">
+        <h1 class="title dark-blue">Charlie Giorgania</h1>
+        <div class="contents">
+          <p class="paragraph dark-blue">
+            Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work.
+          </p>
+          <p class="paragraph dark-blue">
+            If you like what you see and have a project you need coded, don't hesitate to contact me.
+          </p>
+          <button class="collab-button bg-dark-blue white">Start collaboration</button>
+        </div>
+      </div>
+      <div>
+        <div class="thur-background-img css-background-img"></div>
       </div>
+    </section>
+  </div>
+  
+  <!--Works container 1-->
+  <secion class="section-container">
+    <div class="section-sub">
+      <div><h1 class="title dark-blue">Projects</h1></div>
+    </div>
+
+    <div>
+      <img class="img-laptop1" src="images/laptop-img.png" alt="This is an image of a person using a laptop." width="320" height="186">
+    </div>
```

Comment:
- [ ] To maintain a **correct display of your works section** and ensure that your project is consistent and responsive, I suggest using `display: grid` for the parent container of your projects.

  Using `display: grid` **can help you easily align and position your project cards in a [grid format](https://www.w3schools.com/css/css_grid.asp)**, while maintaining their responsiveness on different screen sizes. This is a good practice to follow, as it can enhance the overall user experience of your portfolio page. 😉